_posts/2023-04-12-vscode-quarkus-1.13.0-released.adoc

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: 'New Features for Qute Templating Engine Support in Quarkus Tools for Visual Studio Code 1.13.0'
+synopsis: Quarkus Tools for Visual Studio Code version 1.13.0 brings many new features in its Qute Templating Engine support.
+date: 2023-04-04
```

Comment:
Let's target for a publication tomorrow: you need to update this date and the date of the file accordingly.